vllm/model_executor/models/interfaces.py

```diff
@@ -706,9 +719,30 @@ def get_generation_prompt(cls, audio: np.ndarray,
         ...
 
     @classmethod
-    def validate_language(cls, language: str) -> bool:
-        """Check if the model supports a specific ISO639_1 language."""
-        ...
+    def _other_languages(cls) -> Mapping[str, str]:
+        # other possible language codes from the whisper map
+        return {
+            k: v
+            for k, v in LANGUAGES.items() if k not in cls.supported_languages
+        }
+
+    @classmethod
+    def validate_language(cls, language: Optional[str]) -> Optional[str]:
+        if language is None or language in cls.supported_languages:
```

Comment:
can you add docstrings to interface methods?

Reply:
Resolved in https://github.com/vllm-project/vllm/pull/21833/commits/cd4e0a1df22c3ad5435c71c00c5de8653a55ce2e